practice-two/src/services/customerService.js

```diff
@@ -1,12 +1,58 @@
-// import { BASE_URL, PATH } from "@constants";
+import { BASE_URL, PATH } from '@constants';
+import { mutate } from 'swr';
 
+// Get customer list
 const getAllCustomerService = async (url) => {
```

Comment:
@datdragon777 Why you don't using `request` method for get all customers.

Reply:
If I use request method to get all customers, the result will return a object with data and error
```
const request = async (path, method, data) => {
  const url = `${BASE_URL}/${path}`;
  try {
    const response = await fetch(url, {
      method,
      headers: {
        'Content-Type': 'application/json',
      },
      body: JSON.stringify(data),
    });

    if (response.ok) {
      mutate(url);
      return { data: await response.json(), error: null };
    } else {
      const errorData = await response.json();
      return { data: null, error: errorData };
    }
  } catch (error) {
    return { data: null, error };
  }
};
```

When I use a SWR to fetch API, the SWR supports data, error and isLoading method to handle calling API
If I do the calling API like function above, I must call `data.error`
It's not right when I do that with data, I want data just return the result of list
And error of SWR will only return a error